.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,19 @@
+Thanks for filing an issue! Please remove any top-level headers that do not apply.
+
+# Problem or Error
+Please characterize the problem concisely and precisely. You may want to include error message output, other terminal output, your environment characteristics, and a way to replicate the issue. Let us know what solutions you have tried.
+
+# Question
+Please ask the question with clear context and any potential answer directions you can think of.
+
+# Variable to add: name
```

Comment:
I think the parameters/variables to add may be removed since they are not relevant to OG-ZAF.  Any new model parameters or variables would be defined in OG-Core.